reviews/incubation-crossplane.md

```diff
@@ -0,0 +1,250 @@
+With these versions of Crossplane, the project is now stable and ready to be used in production
+scenarios. This is further validated by project adopters successfully running Crossplane in
+production environments. The core APIs have been moved to v1 (stable) and future enhancements and
+improvements are intended to be done in a backwards compatible way with a clear upgrade path.  We do
+not anticipate any further breaking changes in subsequent minor and patch releases.
+
+With Crossplane reaching maturity, we are seeing end-users adopt Crossplane and find success running
+it in their production environments. We consistently hear from users that have built their own
+custom platforms APIs to enable self-service provisioning for their developers and accelerate the
+delivery of their software to production. The community growth has been quite encouraging and we are
+pleased to be able to include specific details about some of their adoption success in the sections
+below.
+
+The Crossplane community has expanded its collaboration with a rich ecosystem of partners, building
+relationships and combining development efforts with a broader set of cloud providers and projects
+in meaningful ways:
+
+* IBM developed and released the [first version of their Crossplane provider for IBM
+  Cloud](https://developer.ibm.com/blogs/ibm-joins-the-crossplane-community/), enabling the
+  provisioning and management of 85+ hosted services from the IBM Cloud Catalog.
+* Engineering teams from both [AWS and
+  Azure](https://blog.crossplane.io/accelerating-crossplane-provider-coverage-with-ack-and-azure-code-generation-towards-100-percent-coverage-of-all-cloud-services/)
+  have joined forces with the Crossplane community to share their code generation pipelines,
+  allowing the Crossplane providers for AWS and Azure to increase their coverage and more easily
+  stay updated with changes in their cloud offerings.
+* RedHat partners with Crossplane for infrastructure provisioning, with a focus on hybrid scenarios.
+  Crossplane has been available on [operatorhub.io](https://operatorhub.io/operator/crossplane)
+  since it’s v0.9 release.
+* GitLab has integrated Crossplane into [GitLab
+  ADO](https://docs.gitlab.com/ee/user/clusters/crossplane.html) so users can easily provision and
+  manage infrastructure from their Kubernetes clusters. Crossplane has been shipped with every
+  GitLab release since version 12.5.
+* Packet/Equinix developed the [Crossplane Provider for Equinix
+  Metal](https://feedback.equinixmetal.com/changelog/crossplane-provider-for-equinix-metal), as well
+  as
+  [provider-ssh](https://www.youtube.com/watch?v=CxwmrKi1jtM&list=PL510POnNVaaYFuK-B_SIUrpIonCtLVOzT&index=6)
+  as a way to provision, manage, and run scripts against bare metal instances and VMs
+* The following are examples of ecosystem projects that have collaborated and integrated with
```

Comment:
Hi @jbw976 . Just want to point out two more ecosystem use cases if you want to add them here:

- KubeVela project uses Crossplane as cloud resource provider to provision infrastructure resources and bind to applications.
- Alibaba internal platforms have adopted Crossplane to model and manage infrastructure resources as well

Reply:
Thank you for those details and reminder @hongchaodeng, I will include those in the update tonight.  Thank you very much!

Reply:
https://github.com/displague/provider-cloudinit is moving forward as a way to manage userdata, it is in a better state than provider-ssh.